site/docs/pages/identity/avatar.mdx

```diff
@@ -1,10 +1,11 @@
 import { Avatar, Badge, Identity } from '@coinbase/onchainkit/identity';
 import App from '../../components/App';
+import { base } from 'viem/chains';
 
 # `<Avatar />`
 
-The `Avatar` component is used to display ENS avatar associated with Ethereum addresses.
-When an ENS avatar is not available, it defaults to blue color avatar.
+The `Avatar` component is used to display ENS or Basenames avatar associated with Ethereum addresses.
+When an avatar is not available, it defaults to blue color avatar.
```

Comment:
We should mention also the default version for Basename.

Reply:
The default version for Basename is the same as ENS, therefore I've updated this to:

`Address without an ENS or Basenames avatar defaults to a plain avatar:`

Reply:
> When an avatar is not available, it defaults to blue color avatar.

This appears to be out of date.